mmv1/third_party/terraform/services/orgpolicy/resource_org_policy_policy_test.go

```diff
@@ -458,3 +458,108 @@ func testAccCheckOrgPolicyPolicyDestroyProducer(t *testing.T) func(s *terraform.
 		return nil
 	}
 }
+func TestAccOrgPolicyPolicy_EnforceParameterizedMCPolicy(t *testing.T) {
+        t.Parallel()    
+        
+        context := map[string]interface{}{
+                "org_id":        envvar.GetTestOrgFromEnv(t),
+                "random_suffix": acctest.RandString(t, 10),
+        }               
+
+        acctest.VcrTest(t, resource.TestCase{
+                PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+                ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+                CheckDestroy:             testAccCheckOrgPolicyPolicyDestroyProducer(t),
+                Steps: []resource.TestStep{
+                        {
+                                Config: testAccOrgPolicyPolicy_EnforceParameterizedMCPolicy(context),
+                        },      
+                        {       
+                                ResourceName:            "google_org_policy_policy.primary",
+                                ImportState:             true,
+                                ImportStateVerify:       true,
+                                ImportStateVerifyIgnore: []string{"name", "spec.0.rules.0.condition.0.expression"},
+                        },
+                },      
+        })                      
+}
+func testAccOrgPolicyPolicy_EnforceParameterizedMCPolicy(context map[string]interface{}) string {
+        return acctest.Nprintf(`
+resource "google_org_policy_policy" "primary" {
+  name   = "projects/${google_project.basic.name}/policies/constraints/compute.managed.restrictDiskCreation"
+  parent = "projects/${google_project.basic.name}"
+
+  spec {
+    rules {
+      enforce = "TRUE"
+      parameters {
```

Comment:
See https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_routine#example-usage---bigquery-routine-tvf for how a JSON field is specified!